### DISCUSSION TITLE
List full paths in `typst fonts`

### DISCUSSION THREAD

**Discussion by daeh** [typst/typst#4948] [Ideas]
It would be great to know which font files typst is actually using. At the moment I'm having trouble determining if some of the fonts listed (e.g. "New Computer Modern") are using my System font files or the version packaged with with compiler.

**Comment by ssotoen** [comment]
The priority is `--font-path` > system fonts > packaged fonts.

**Reply by daeh** [reply]
Yes, thanks, I do know that. The paths would still be useful, especially for debugging, and for when the displayed font name does not match the file name. (I'm finding it difficult to tell whether Typst is actually using the OTF in my system fonts or not and I haven't found a way to actually prove to myself that it is using the system font. It seems like just printing the paths alongside the font names would remove all of the ambiguity.)
